docs/dev_guide/contents/newcomers.rst

```diff
@@ -10,7 +10,7 @@ The people who help develop or contribute to SunPy are varied in ability and exp
 
 If you have any questions, comments, or just want to say hello, we have an active `chat room`_, a `mailing list`_ and a `community forum`_.
 
-.. _chat room: https://app.element.io/#/room/#sunpy:openastronomy.org
+.. _chat room: https://app.element.io/#/register
```

Comment:
As far as I can tell, this is the correct link, before the change.

Reply:
But as far as I've experienced, newcomers won't be able to create an account by following that link https://app.element.io/#/room/#sunpy:openastronomy.org .  They have to first create an account in matrix main server and then join the sunpy room.

Reply:
Newcomers will be able to create a account following that link, it will allow them to register on the main element server. 

The room loads, the sign up button appears in the middle for me. What is the problem that I am not understanding?